text/0118-search-api.md

```diff
@@ -892,7 +894,32 @@ It's useful when more control is needed than offered by the built-in highlightin
 
 - 🔴 Sending a value with a different type than `Boolean` or `null` for `showMatchesPosition` returns an [invalid_search_show_matches_position](0061-error-format-and-definitions.md#invalid_search_show_matches_position) error.
 
-#### 3.1.17. `matchingStrategy`
+#### 3.1.17. `showRankingScore`
+
```

Comment:
The experimental is only `showRankingScoreDetails`?

Reply:
Yes that's correct.

`showRankingScore`: stable

`showRankingScoreDetails`: experimental